analyzers/rspec/cs/S6803.json

```diff
@@ -1,7 +1,7 @@
 {
   "title": "Parameters with SupplyParameterFromQuery attribute should be used only in routable components",
   "type": "CODE_SMELL",
-  "status": "ready",
+  "status": "deprecated",
```

Comment:
I wrote also on RSPEC PR, shouldn't we remove the rule from "Sonar Way" profile?

Reply:
Yes! Thanks for catching this.